src/mocks/msw/handlers.ts

```diff
@@ -58,6 +58,18 @@ export const handlers = [
     mswEndpoint('/api/v1/workspaces/active'),
     () => new HttpResponse(null, { status: 204 })
   ),
+  http.put(mswEndpoint('/api/v1/workspaces/:workspace_name'), () =>
+    HttpResponse.json(
+      {
+        name: 'foo',
+        config: {
+          custom_instructions: '',
+          muxing_rules: [],
+        },
+      },
+      { status: 201 }
+    )
+  ),
```

Comment:
so the BE Pr is replacing the APIs for update workspace config with a single update API, right?